test/command/suite/normalizers/nfkc100/unify_to_katakana/etc.test

```diff
@@ -0,0 +1,5 @@
+normalize \
+  'NormalizerNFKC100("unify_to_katakana", true, \
+                     "report_source_offset", true)' \
+  "ゔゕゖゝゞゟ" \
+  WITH_CHECKS|WITH_TYPES
```

Comment:
It seems that "ゔぁゔぃゔゔぇゔぉ" test case is missing. Is it needless?

Reply:
Yes. "ゔぁゔぃゔゔぇゔぉ" test case is needless.
Because, this case has already tested by a.test and etc.test.

In my opinion, this option normalize hiragana to katakana.
Therefore, we only have to test "ゔ" to "ヴ" and "ぁぃぅぇぉ" to "ァィゥェォ".

Reply:
Ah, I've got it. It is reasonable.